domains/logistics/configure-organization/README.md

```diff
@@ -2,56 +2,60 @@
 
 ## Description
 
-The FL Logistics has a depot in Lyon and another in Saint-Étienne. Each depot has one _receiving_ and one _picking_ setting. Each _receiving_ setting has one forklift, one pallet jack, and one barcode reader. Each _picking_ setting has one forklift, one pallet jack, and one label printer. Only one employee can be associated with the usage of one setting, but the same employee can be associated with the usage of multiple settings.
+The FL Logistics has a depot in Lyon and another in Saint-Étienne. Each depot has one _receive_ and one _pick_ setting.
 
-Forklifts are capable of lifting up and lifting down materials on pallets to/from high shelves, as well as moving them from one location to another. Pallet jacks are capable of lifting up and lifting down materials on pallets placed on the floor and move them from one location to another. They both are used for loading and unloading trucks by moving materials in the _receiving_ and the _picking_ settings.
+In the _receive_ setting, agents have access to equipment that can handle (i.e., lift and move) pallets, and list the products in a pallet based on barcode.
 
-The barcode reader is capable of reading barcodes identifying materials on a pallet. The label machine is capable of printing labels with barcode to identify the materials on a pallet.
+In the _pick_ setting, agents have access to equipment that can handle (i.e., lift and move) pallets, and print barcode labels identifying the list the products in a pallet.
```

Comment:
identifying the list _of_ products in a pallet?

Reply:
We do not represent the individual products and this is not the objective here. Would it work if I change the sentence to

> In the _pick_ setting, agents have access to equipment that can handle (i.e., lift and move) pallets, and print barcode labels identifying the **pallets' content**.

Reply:
Replaced by

In the _pick_ setting, agents have access to equipment that can handle (i.e., lift and move) pallets and print labels with barcode identifying the pallets' content.